blueprint/index.md

```diff
@@ -84,9 +84,31 @@ The integration provides the following examples of Architect flows:
 
 ### HRIS-Get-Agents flow
 
-This HRIS provides a list of all agent records, including their corresponding IDs/keys and email addresses. Genesys Cloud workforce management requires this data to retrieve time-off balances and insert time-off requests for an agent.
+The purpose of this workflow is to to provide unique ids of agents in external HRIS system together with their emails, according to which these agents can be mapped to the user records in Genesys Cloud. 
+
+This workflow is expected to return a list of emailIds and associated externalIds of the agents from external HRIS system that is to be synchronized with Genesys users. 
+
+The number of records in both lists should match. The list order is important as the system will use the order to match elements in list of ids with the list of emails.
+
+The email ID of a given agent configured in external HRIS system should match to that of the user defined in Genesys and that is the key based on which the agents in two systems are integrated.
+
+The flow does not have need a input parameter and the out of the workflow would be a list of emailId and externalId both of which are string data types. Below is a table showing that information.
+
+Note that a workflow variable can only have up to 2000 entries maximum and hence to facilitate that there are 25 buckets of emails and externalIds provided to send up 50K agent details.
+The indexed of additional buckets start from 1 through 24 in addition to the initial bucket. 
+
+The workflow is assigned to the WFM Integration configuration property of "User Account IDs" that has a description of "An architect workflow to retrieve a list of users from HRIS". This will ensure the workflow
+is triggered as part of scheduled agent synchronization process.
+
+| Name              |   Type   |             Data Type              | Notes                                                                                                |
+|:------------------|:-------- |:----------------------------------:|:-----------------------------------------------------------------------------------------------------|
+| Flow.statusCode   | Output   |      HTTP status code <br/>Integer      | Must be set for all workflows                                                                        
+| Flow.status       | Output   |               String               | Must be set for all workflows.<br> Possible values are Running, Error, Terminated, Completed, Failed 
```

Comment:
So far I have seen just Error and Completed. 
Running is incorrect, because if it returned already, how can it be Running.
What is Terminated and Failed for and when they supposed to be used instead of Error? If we can document how to use them, then let's document it. Otherwise, let's remove them.

Reply:
Terminated and failed. Never seen it. Running is not incorrect because you could poll and automate if not done will set status to running. I am not sure if we should remove from documenting since these are possible values documented in Automate code I think

Reply:
public enum FlowRunningState
{

    Running,                // This (obviously) only would apply to active flows
    Error,                  // This only would apply to active flows

    Terminated,             // The flow is not running anymore, it has been permanently terminated
    Completed,              // The flow is not running anymore, it has completed successfully
    Failed,                 // The flow is not running anymore, it has permanently failed
}